src/sage/rings/function_field/drinfeld_modules/finite_drinfeld_module.py

```diff
@@ -369,18 +413,19 @@ def _frobenius_charpoly_gekeler(self, var='X'):
             sage: phi = DrinfeldModule(A, [z, 0, z])
```

Comment:
Can you replace Line 109 with

```

            ::

```

Reply:
The Rubric line? Not sure if this is what you want so I'll leave this for now.

Reply:
I apologize, this was not clear at all by me. Apparently, what I meant was Line 418. I tagued you in the relevant comment.